bigbluebutton-html5/imports/ui/components/modal-consent/utils/getValueFromKeyObj.js

```diff
@@ -0,0 +1,11 @@
+export default function getValueFromKeyObj(keyObj, obj) {
```

Comment:
Consider to remove this function and use short-circuit evaluation with || instead